app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -368,9 +370,11 @@ public void showBadImagePopup(Integer errorCode) {
         String errorMessageForResult = getErrorMessageForResult(getContext(), errorCode);
         if (!StringUtils.isBlank(errorMessageForResult)) {
             DialogUtil.showAlertDialog(getActivity(),
-                    getString(R.string.warning),
+                    getString(R.string.duplicate_image_found),
```

Comment:
I don't think the sting "Duplicate Image Found" suits for this dialog. This dialog is shown for other cases (take a look at the `ImageUtils#getErrorMessageForResult` method to get an idea). May be "Problems found in image" would be an appropriate title.

Reply:
@macgills could I get a definitive answer on what to name these buttons please? I don't want to just keep committing...

Reply:
it depends on the dialog for `showDuplicatePicturePopup` "Duplicate Image" makes sense, this is `showBadImagePopup` which has various pieces of text
![image](https://user-images.githubusercontent.com/3358282/77167168-67b4a580-6aad-11ea-8b06-f0ea2703c3d8.png)


@sivaraam is correct here

Reply:
My bad, I realised you guys meant I used the same title for both methods. All done now!